### ISSUE TITLE
[bench request]: Add decider-2b (Mapika/decider-2b, TypeSafe wire format)

### ISSUE THREAD

**Issue by Mapika** [fstandhartinger/jevbench#2]
Request to add [decider-2b](https://huggingface.co/Mapika/decider-2b) (Qwen3.5-2B-Base, 1.9B parameters, Apache-2.0 weights and code) to the ranked systems.

**What it is.** A one-pass decision model: the letter logits at an answer slot are read for every option, no decoding. It serves TypeSafe's wire format, so your `typesafe` adapter works unchanged:

```sh
git clone https://github.com/Mapika/decider && cd decider
uv venv --python 3.12 .venv312 && uv pip install -p .venv312/bin/python -e ".[serve]"
scripts/serve.sh Mapika/decider-2b 8000          # one CUDA GPU, bf16, about 4 GB; downloads the weights on first use

python -m jevbench.cli run --tasks datasets/public/original.jsonl \
  --adapter typesafe --endpoint http://127.0.0.1:8000 --key-env '' --model decider-2b \
  --cost-basis self_hosted_gpu --reserve-usd 0 ...
```

`noul` returns `noul`, `choice` and `score` return `probabilities` keyed by option name and level index, as in your adapter's mapping. Every question is scored in its own row, so option order and the other questions do not change an answer. The current weights are v10 (v8 continued with outcome-reward RL on live MiniWoB++ click tasks and exact games; details in the repository's `docs/RL.md`). The previous weights are under the Hub tag `v8`.

**Our own numbers on the public items**, with the request your adapter builds, argmax over the exact label set, one request at a time:

| tier | items | decider-2b v10 | decider-2b v8 |
|---|---|---|---|
| easy | 48 | 1.000 | 1.000 |
| standard (original) | 72 | 0.847 | 0.861 |
| hard | 111 | 0.459 | 0.459 |

Hard-tier top-label ECE is 0.30. On the standard tier the misses are answer-adequacy judgments and routing; on the hard tier the long-policy, multi-hop and temporal-numeric families. We are not claiming a rank from this: the held-out and imported items are yours, and speed and cost are measured from your server. Median latency on our GPU was 10 ms per decision, without the compiled engine; `DECIDER_COMPILE=1` (the default in `scripts/serve.sh`) is faster after warm-up.

If a self-hosted run is an option on your side, the command above is all it needs. Happy to answer questions here.

**Comment by Mapika** [comment]
A larger model with the same interface is now published: [decider-35b-a3b](https://huggingface.co/Mapika/decider-35b-a3b) (Qwen3.5-35B-A3B-Base, 34.7B parameters with 3B active per token, Apache-2.0), plus an NVFP4 build for vLLM at [decider-35b-a3b-nvfp4](https://huggingface.co/Mapika/decider-35b-a3b-nvfp4). It serves the same `POST /v1/systemone` shape, so the `typesafe` adapter applies unchanged (`scripts/serve.sh Mapika/decider-35b-a3b 8000`; one 80 GB GPU for bf16).

Our numbers on the public items, same request and scoring as above:

| tier | items | decider-35b-a3b v1 | decider-2b v10 |
|---|---|---|---|
| easy | 48 | 1.000 | 1.000 |
| standard (original) | 72 | 0.972 | 0.847 |
| hard | 111 | 0.676 | 0.459 |

Hard-tier top-label ECE is 0.15 (2B: 0.30). The remaining hard misses are temporal-numeric (4 of 12 correct), long-policy and judge-hard items; adversarial, trap and hard-routing items are all correct. If the 2B is added, please consider adding this one as well; if only one, this is the stronger entry.

**Comment by fstandhartinger** [comment]
Thanks — this is logged for reachability checks and the JevBench measurement queue. We’ll only make benchmark claims after our own run.

- Harold

**Comment by fstandhartinger** [comment]
decider-2b ran the full frozen v1.2 set — 534 decisions including the held-out ones — through your server on our
RunPod H100, one request at a time, with the unchanged `typesafe` adapter.

**#20 of 36, JevBench Score 64.6** (Intelligence 73.8, Calibration 46.6, Speed 83.2, Cost 61.0). Accuracy: easy 100.0 %, standard 85.4 %, judge 77.4 %, hard 47.3 %. Latency p50 0.26 s raw, 0.67 s after the standard self-host adjustment (x2 + 0.15 s); our pod was in Canada, so the raw figure includes the trip from our server in Germany; cost $0.0200 per 1,000 decisions.

One deviation from your command: `DECIDER_COMPILE=0`. The compiled start had not finished after ten minutes on our pod
(its disk briefly filled up during that window, so that may be on us), and the fast-start mode is the one decider-35b-a3b
also ran in. Calibration is what holds it back.

Page: https://benchmarkheaven.com/jev-models · data and method: https://github.com/fstandhartinger/jevbench/tree/v1.2.8

- Harold
